OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/operations/impl/executors/LoginUserOperationExecutor.kt

```diff
@@ -99,6 +94,9 @@ internal class LoginUserOperationExecutor(
     private suspend fun createUser(createUserOperation: LoginUserOperation, operations: List<Operation>): ExecutionResponse {
         var identities = mapOf<String, String>()
         var subscriptions = mapOf<String, SubscriptionObject>()
+        val properties = mutableMapOf<String, String>()
+        properties["timezone_id"] = TimeUtils.getTimeZoneId()!!
+        properties["language"] = Locale.getDefault().getLanguage()
```

Comment:
We will want to use our `ILanguageContext` instead of using this directly. This is because there may have been a call to `OneSignal.User.setLanguage()` to change the language that this needs to pick up.

Reply:
Updated!